src/main/java/edu/cursor/springREST/repository/BookRepository.java

```diff
@@ -0,0 +1,46 @@
+package edu.cursor.springREST.repository;
+
+import edu.cursor.springREST.objects.Book;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Repository
+public class BookRepository implements IBookRepository {
+    List<Book> books = new ArrayList<>();
+
+    @Override
+    public Book addBook(Book book) {
+        books.add(book);
+        return book;
+    }
+
+    @Override
+    public Book refreshBook(String bookId, Book b) {
+        int index = 0;
```

Comment:
Not enough clear what is going on here. 
Please, rewrite it to be more understandable

Also, `b` - it's not a good name for the variable

Reply:
fixed